### PR TITLE
Autopark Level01

### PR DESCRIPTION
**Pull request by dzmitri-litvinov**
Create VehicleType class

### CODE REVIEW

File: autopark/VehicleType.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+        public override string ToString()
+        {
+            return $"{typeName} {taxCoefficient}";
+        }
+
+        public void ChangeTaxCoef(float n)
+        {
+            taxCoefficient = n;
+        }
```
Comment (UNDogMan) [inline autopark/VehicleType.cs]: You don't need these method. Use Property instead.

File: autopark/VehicleType.cs [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    class VehicleType
+    {
+        string typeName;
+        float taxCoefficient; //or float taxCoefficient { get; set; }
```
Comment (UNDogMan) [inline autopark/VehicleType.cs]: Don't leave comments with code.

File: autopark/VehicleType.cs [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace autopark
+{
+    class VehicleType
+    {
+        string typeName;
```
Comment (UNDogMan) [inline autopark/VehicleType.cs]: You don't have access to this field. Create Property for that field or replace in with Auto Property like that.
```suggestion
        public string TypeName { get; set; }
```

File: autopark/VehicleType.cs [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+        float taxCoefficient; //or float taxCoefficient { get; set; }
+
+        public float TaxCoefficient
+        {
+            get { return taxCoefficient; }
+            set { taxCoefficient = value; }
+        }
```
Comment (UNDogMan) [inline autopark/VehicleType.cs]: Use Auto Property, if you don't need special logic for field. Auto Properties makes code more readable and compacted.

**Review by UNDogMan — CHANGES_REQUESTED** [review]
I leave some comments for you. Read them and make fixes.

File: autopark/VehicleType.cs (hunk 1 above)
Comment (dzmitri-litvinov) [reply autopark/VehicleType.cs]: image
Done

File: autopark/VehicleType.cs (hunk 3 above)
Comment (dzmitri-litvinov) [reply autopark/VehicleType.cs]: image
Done

File: autopark/VehicleType.cs (hunk 4 above)
Comment (dzmitri-litvinov) [reply autopark/VehicleType.cs]: image
Done

File: autopark/VehicleType.cs (hunk 2 above)
Comment (dzmitri-litvinov) [reply autopark/VehicleType.cs]: Done

**Review by UNDogMan — APPROVED** [review]
LGFM

**Pull request merged**